azure-pipelines/dotnet.yml

```diff
@@ -20,10 +20,17 @@ steps:
   displayName: ⚙ Update pipeline variables based on build outputs
   condition: succeededOrFailed()
 
-- powershell: azure-pipelines/artifacts/_pipelines.ps1 -ArtifactNameSuffix "-$(Agent.JobName)" -Verbose
-  failOnStderr: true
-  displayName: 📢 Publish artifacts
-  condition: succeededOrFailed()
+- ${{ if eq(variables['system.collectionId'], '011b8bdf-6d56-4f87-be0d-0092136884d9') }}:
```

Comment:
Does Azure yaml support comments to explain what these guids are?

Reply:
Yes. This guid appears in many yaml files though, so I'm not sure where it would be best to comment on it. It's the devdiv account guid.